pkg/exthttp/transport.go

```diff
@@ -25,3 +44,25 @@ func NewTransport() *http.Transport {
 		ExpectContinueTimeout: 1 * time.Second,
 	}
 }
+
+// Default Transport function that creates a http.Transport with default settings.
```

Comment:
There's a spurious space here. Also, the default settings? I thought that's what NewTransport does :P I suppose this should say "with specified settings". Plus, maybe we could reuse this in `NewTransport()`?

Reply:
By that you mean to call the ```DefaultTransport()``` from ```NewTransport()``` with the default settings?